README.md

```diff
@@ -42,6 +42,12 @@ For more information on how to use Stripe.js once it loads, please refer to the
 [accept a payment](https://stripe.com/docs/payments/accept-a-payment) with
 Stripe.
 
+If you have deployed
+[CSP](https://developer.mozilla.org/en-US/docs/Web/Security/CSP), please refer
+to the
+[Security Guide](https://stripe.com/docs/security/guide#content-security-policy)
+to allow Stripe.js in your directives.
```

Comment:
(total nit) re-reading this a second time, it felt weird that the action of "visiting the security guide" doesn't lead to directives being enabled; just lets a user _learn how_ to enable directives. How about

> If you have deployed a [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/Security/CSP), make sure to [include Stripe.js in your directives](https://stripe.com/docs/security/guide#content-security-policy).